db/migrate/20171009173946_remove_configuration_templates_table.rb

```diff
@@ -0,0 +1,12 @@
+class RemoveConfigurationTemplatesTable < ActiveRecord::Migration[5.0]
+  def change
+    drop_table :configuration_templates do |t|
```

Comment:
COOL I've never seen a drop_table with a block like this so that `change` will work 👍